pkg/ccl/crosscluster/physical/stream_ingestion_frontier_processor.go

```diff
@@ -392,7 +392,6 @@ func (sf *streamIngestionFrontier) maybeUpdateProgress() error {
 		return err
 	}
 
-	sf.metrics.JobProgressUpdates.Inc(1)
```

Comment:
per our discussion today, i think this metric is now useful ;)

Reply:
For what?

Reply:
are you thinking of checkpoint events? that's still here and isn't going anywhere.

Reply:
My takeaway from our discussion yesterday was that we wanted to have a count which increments every time we bump the global resolvedTime, which is what this metric _almost_ does. (We need to bump it only when the resolvedTime advances). 

Are you referring to the `ResolvedEvents` metric? That gets incremented per processor, and would not signal when the global resolvedTimestamp is advancing or not.

Reply:
I think we just said we wanted to know if we're getting checkpoints or not, which is what `ResolvedEvents` is. It is per-node, which is actually pretty nice since if it is increasing not increasing anywhere vs flat on just a single node that tells you something (e.g look for a stuck RF).

Reply:
hrm i dont think we're on the same page here bug i'll approve to unblock you for now.